src/EnergyPlus/WaterCoils.cc

```diff
@@ -3385,8 +3329,8 @@ namespace WaterCoils {
 		AirInletCoilSurfTemp = PsyTsatFnHPb( EnthSatAirCoilSurfaceEntryTemp, OutBaroPress );
 
 		// Calculate outlet air temperature and humidity from enthalpies and surface conditions.
-		TotWaterCoilLoad = AirMassFlow * ( EnthAirInlet - EnthAirOutlet );
-		OutletWaterTemp = WaterTempIn + TotWaterCoilLoad / max( WaterMassFlowRate, SmallNo ) / Cp;
+		TotWaterCoilLoad = WaterCoil( CoilNum ).InletAirMassFlowRate * ( EnthAirInlet - EnthAirOutlet );
+		OutletWaterTemp = WaterTempIn + TotWaterCoilLoad / max( WaterCoil( CoilNum ).InletWaterMassFlowRate, SmallNo ) / Cp;
 
```

Comment:
@Myoldmopar I thought you said you pushed the fix for this. These two lines are still making a change that they shouldn't be. After I finish this pass I'll refresh and make sure.